community-operators/okd-windows-machine-config-operator/0.0.1/okd-windows-machine-config-operator.0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,229 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+    categories: "OpenShift Optional"
+    certified: "false"
+    containerImage: quay.io/openshift-windows/okd-windows-machine-config-operator:4.6-1d646e8
+    createdAt: 2020-09-01T11:59:59Z
+    description: An operator which allows for Windows workloads to be run within OKD
+    operatorframework.io/cluster-monitoring: "true"
+    operatorframework.io/suggested-namespace: windows-machine-config-operator
+    repository: https://github.com/openshift/windows-machine-config-operator
+    support: ""
+  name: okd-windows-machine-config-operator.v0.0.1
+  namespace: windows-machine-config-operator
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions: {}
+  description: |-
+    # OKD Windows Machine Config Operator
+
+    ## WARNING
+    Community distribution of the Windows Machine Config Operator. This is an alpha build, and is not meant for
+    production workloads. Issues with this distribution of WMCO are not supported.
+
+    Please ensure that when installing this operator you choose the starting version which lines up with the version of
+    OKD you are using.
+
+    Upgrading the OKD cluster while running this operator is not supported. When upgrading OKD, remove all
+    Windows machinesets and uninstall this operator. Once the OKD upgrade process has completed, you may reinstall
+    the operator, and recreate all Windows machinesets.
+
+    ## Documentation
+
+    The Windows Machine Config Operator configures Windows Machines into nodes, allowing Windows workloads to be run
+    within OKD clusters. For more information please looks at the repository [README](https://github.com/openshift/windows-machine-config-operator/blob/master/README.md)
```

Comment:
I suggest using `enabling Windows workloads on OKD`. I suggest not linking to the README as that is tailored to running the operator from source. You should pick the required paragraphs from there that just pertain to running the operator, like the MachineSet section etc.